README.md

```diff
@@ -33,13 +50,23 @@ $ mvn clean compile
 $ mvn package
 ```
 
-This will create a `target` directory in which you will find the executable jar file that you can use for the 
+This will create a `target` directory in which you will find the executable jar file that you can use.
 
 ### How to run IRI 
 
 #### Locally
 
-Running IRI is pretty simple, and you don't even have to run it under admin rights. Below is a list of command line options. Here is an example script:
+Running IRI is quick and easy, and you can usually run it without admin rights.
+Below is a list of command line options.
+At a minimum you must provide one of:
+
+- the port — e.g., '`-p 14265`'
+- the location of an ini config file — e.g., '`-c iota.ini`' 
+- the default config ini file must exist **and also** contain the port setting — e.g.,'`PORT = 14265`'.
```

Comment:
I think "must" is a strong word here. I can run IRI without it. Also, if PORT is missing from the ini then IRI can run if it is given via CLI.